app/resources/v1/pacbio/pool_resource.rb

```diff
@@ -13,6 +10,44 @@ module Pacbio
     # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
     # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package
     # for the service implementation of the JSON:API standard.
+    # This resource represents a Pacbio Pool and can return all pools, a single pool or multiple
+    # pools along with their relationships.
+    #
+    # ## Filters:
+    #
+    # * sample_name
+    # * barcode
+    #
+    # ## Primary relationships:
+    #
+    # * tube
+    # * used_aliquots
+    # * primary_aliquot
+    # * requests
+    # * libraries
+    #
+    # ## Relationship trees:
+    #
+    # * used_aliquots.tag.tag_set
```

Comment:
I think these relationship trees can be as deep as you want. For example you could go used_aliquots.source.primary_aliquot.tag... etc depending on what information you want. Might be worth having a higher level explanation about these trees / jsonapi somewhere in our docs.